### ISSUE TITLE
Interpolation with PyTorch F.interpolate module yield horse in binary mask

### ISSUE THREAD

**Issue by RRobert92** [SMLC-NYSBC/TARDIS#15]
Downsampling for 3D (possibly also 2D) with mode 'trilinear' yield holes during upsampling:
3D
`F.interpolate(torch.Tensor(image)[None, None, :], scale_factor=scale, mode='trilinear', align_corners=False).cpu().detach().numpy()[0, 0, :]`
2D
`F.interpolate(torch.Tensor(image)[None, None, :],
                              scale_factor=scale,
                              mode='bicubic',
                              align_corners=False).cpu().detach().numpy()[0, 0, :]`
